src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -524,6 +528,32 @@ export class PatientViewPageStore {
         }
     }, false);
 
+    readonly uncalledMutationData = remoteData({
+        await: () => [
+            this.samples
+        ],
+        invoke: async() => {
+            const geneticProfileId = this.uncalledMutationGeneticProfileId;
+            if (geneticProfileId) {
+                let uncalledMutations = await client.fetchMutationsInGeneticProfileUsingPOST({
+                        geneticProfileId,
+                        mutationFilter: {
+                            sampleIds: this.samples.result.map((sample: Sample) => sample.sampleId)
+                        } as MutationFilter,
+                        projection: "DETAILED"
+                    });
+                for (const mutation of uncalledMutations) {
+                        // mutationStatus is hardly used, we can use it to keep
+                        // track of uncalled mutations
+                        mutation.mutationStatus = 'UNCALLED';

```

Comment:
I am not sure if it is a good idea to update/overwrite an API generated data field for custom data purposes...